### PR TITLE
Change implementation of accessible-hide and visually-hidden classes

### PR DESCRIPTION
**Pull request by LidorLapid**
The reason for changing it is to work around an issue in Firefox where absolutely positioned elements affect the outline of their parents

### CODE REVIEW

**Comment by LidorLapid** [comment]
Needs to be browser tested.

File: assets/scss/partials/elements/_sign-posts.scss [hunk 1]
```diff
@@ -77,18 +77,17 @@
...
+                  @include accessible-hide;
+                  margin-top:0px;
+                  margin-bottom:0px;
+                  display: inline-block;
+                  position:absolute;
+                  top:0px;
+                  left:0px;
+                  height:100px;
+                  width:120px;
+                  background-position: center center;
```
Comment (natmunckton) [inline assets/scss/partials/elements/_sign-posts.scss]: Maybe do linting on this code? For example removing px from 0 values, adding a space before the parenthesis open.

File: assets/scss/partials/elements/_sign-posts.scss [hunk 2]
```diff
@@ -77,18 +77,17 @@
...
-                    position:absolute;
-                    top:0px;
-                    left:0px;
-                    height:100px;
-                    width:120px;
-                    @include hide-text;
-                    background-position: center center;
-                    background-size:cover;
-
+                  @include accessible-hide;
```
Comment (natmunckton) [inline assets/scss/partials/elements/_sign-posts.scss]: May need to revert to old sass, as this breaks the image display.

**Comment by LidorLapid** [comment]
Closing this as implementation needs re-thinking. Will log as an issue instead.

**Pull request closed without merging**